examples/user_guide/Plotting_with_Plotly.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we discovered in the [Introduction](Introduction.ipynb), HoloViews allows plotting a variety of data types. Here we will use the sample data module and load the pandas and dask hvPlot API:"
```

Comment:
Same as for matplotlib. One sentence should be enough to link the contents of the notebook to the notebook title.